setup.py

```diff
@@ -3,7 +3,13 @@
 setup(name='Kasa',
     version='0.0.1',
     description='A library of translation and other NLP tools for the Ghanaian language Twi',
+    author='Dr.Paul Azunre',
+    author_email='azunre@gmail.com',
+    license='MIT',
     packages=['Kasa'],
-    install_requires=['gensim >= 3.8.1'],
-    include_package_data=True,
+    install_requires=['gensim == 3.8.1'],
+    include_package_data=True, 
+    include_package_data=True, 
```

Comment:
```include_package_data=True``` should appear only once I believe

Reply:
yeah, I think it was a typo
Let me rectify it

Reply:
The HISTORY.md and LICENSE files shouldn't be moved. Should remain in the base directory

Reply:
Alright
I will do that...Anymore?

Reply:
no that's all.. you may just update the PR

Reply:
Alright

Reply:
pip install kasa works without any issues

Reply:
Joel raised some issues so I will rectify them and update the PR